diff --git a/setup.py b/setup.py
index 4ce17fd..ccae9df 100644
--- a/setup.py
+++ b/setup.py
@@ -62,13 +62,4 @@ setup(
         'License :: OSI Approved :: GNU General Public License v3 (GPLv3)',
         'Programming Language :: Python'
         ]
-)
-
-# Check that the pandoc-secnos script is on the PATH
-if not shutil.which('pandoc-secnos'):
-    msg = """
-          ERROR: `pandoc-secnos` script not found. This will need to 
-          be corrected.  If you need help, please file an Issue at
-          https://github.com/tomduck/pandoc-secnos/issues.\n"""
-    print(textwrap.dedent(msg))
-    sys.exit(-1)
+)
\ No newline at end of file
